.vib/mastodon/goss/goss.yaml

```diff
@@ -0,0 +1,34 @@
+addr:
```

Comment:
I think we can remove the `service.ports` addr checks done here for the subcharts. They are only testing that the subcharts can use different ports for their services, and that is something that should be tested in each chart's test suite.

A more interesting test would be to check that mastodon's config is using those non-default ports to connect to the different services (via mastodon's config file or similar). Similar to this, we could also modify the default auth params of each subchart and check the asset is using them correctly.

Reply:
Mastodon does not have a config file, everything is done via environment variables.

Reply:
In that case, we can check the port-related envs use correct values with the `env` command. That and the check to the `/health` endpoint should be enough.